.github/workflows/publish-to-cocoapods.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish SDK to CocoaPods
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Cocoapods
+        run: gem install cocoapods
```

Comment:
This probably works, but it's using the system Ruby installation. Per https://guides.cocoapods.org/using/getting-started.html this is not recommended.

> However, we recommend not using the system provided Ruby and instead installing a newer Ruby version separate from the system installation

Suggest adding https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby

Reply:
Yep, good call - I'll add that.

Reply:
Done.